README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [Optional] It would be nice to include in your readme a live demo link, in order for other non-programmers visits to have a look at your project and interact with it.